WalletWasabi/Services/WalletService.cs

```diff
@@ -630,6 +634,88 @@ public async Task<Block> GetOrDownloadBlockAsync(uint256 hash, CancellationToken
 					cancel.ThrowIfCancellationRequested();
 					try
 					{
+						// Try to get block information from local running Core node first.
+						try
+						{
+							if (LocalBitcoinCoreNode == null || !LocalBitcoinCoreNode.IsConnected)
```

Comment:
Can be simplified with `LocalBitcoinCoreNode?.IsConnected == true`

Reply:
Do you mean `if(!LocalBitcoinCoreNode?.IsConnected)`?

Reply:
ah right `LocalBitcoinCoreNode?.IsConnected == false` I am not sure `!LocalBitcoinCoreNode?.IsConnected` works

Reply:
ah no wait

Reply:
`LocalBitcoinCoreNode?.IsConnected != true`

Reply:
I'll just leave it like it is for readability:)